JitsiConference.js

```diff
@@ -563,8 +572,8 @@ JitsiConference.prototype._init = function(options = {}) {
             'region', config.deploymentInfo.userRegion);
     }
 
-    // Publish the codec type to presence.
-    this.setLocalParticipantProperty('codecType', this.codecSelection.getPreferredCodec());
```

Comment:
How is backwards compatibility going to work here?

Reply:
The code assumes that older clients that publish only one codec in presence supports only that particular codec - https://github.com/jitsi/lib-jitsi-meet/pull/2296/files#diff-3ed599db7a8e6368e4f4b60ff8eda596e4a49fbd800dbe7128342ff3fc2735cbR131
Since the older clients don't publish the disabled codecs in presence, its safer to assume that the preferred codec is the only supported one.